pkg/test/resource_creator.go

```diff
@@ -48,8 +52,48 @@ func (ctx *Context) GetNamespace() (string, error) {
 	return ctx.namespace, nil
 }
 
+func (ctx *Context) GetOperatorNamespace() (string, error) {
```

Comment:
```suggestion
// GetOperatorNamespace will return an Operator Namespace, if the flag --operator-namespace not be used then it will returna randon value. 
func (ctx *Context) GetOperatorNamespace() (string, error) {
```

Reply:
done